client/flutter/lib/carousel_page.dart

```diff
@@ -81,35 +86,38 @@ class CarouselView extends StatelessWidget {
 
   Widget pageViewIndicator(BuildContext context) {
     var width = MediaQuery.of(context).size.width;
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.center,
-      children: <Widget>[
-        Container(
-          constraints: BoxConstraints(maxWidth: width * 0.75),
-          child: SingleChildScrollView(
-            scrollDirection: Axis.horizontal,
-            child: PageViewIndicator(
-              pageIndexNotifier: pageIndexNotifier,
-              length: this.items.length,
-              normalBuilder: (animationController, index) => Circle(
-                size: 8.0,
-                color: Colors.grey,
-              ),
-              highlightedBuilder: (animationController, index) =>
-                  ScaleTransition(
-                scale: CurvedAnimation(
-                  parent: animationController,
-                  curve: Curves.ease,
+    return Material(
+      elevation: 4.0,
```

Comment:
do we want to have a standardized system for elevation constants, or overkill at this point?

Reply:
4.0 is pretty arbitrary here - if there's a better place to capture this I'm fine with that.

Reply:
fine with this for now!